packages/flame_rive/lib/src/rive_component.dart

```diff
@@ -45,9 +42,8 @@ class RiveComponent extends PositionComponent {
           alignment: alignment,
           artboard: artboard,
         ) {
```

Comment:
```suggestion
        ), super(size: size ?? artboard.toVector2());
```
It should be possible to do something like that instead, so that you don't have to have a body.

Reply:
done